test/integration/verify/controls/azure_redis_cache.rb

```diff
@@ -0,0 +1,16 @@
+resource_group_name = attribute(:resource_group, default: nil)
+inspec_redis_cache_name = attribute(:inspec_redis_cache_name, default: nil)
+
+control 'azure_redis_cache' do
+  describe azure_redis_cache(resource_group: resource_group_name, name: inspec_redis_cache_name) do
```

Comment:
After the control block, we should add the impact and the description. This is how we do in was. Do we have to do the same?

Reply:
it is not mandatory..